custom/icds_reports/ucr/data_sources/dashboard/birth_preparedness_forms.json

```diff
@@ -63,6 +63,19 @@
         },
         "create_index": true
       },
```

Comment:
(nice to have) simplify `state_id` by removing the 'nested'

Reply:
Sure